docs/conferences/dayofevent/README.md

```diff
@@ -47,6 +48,14 @@ In order to provide a better stream to the attendees, based on their proximate d
 ```
 NEXT_PUBLIC_IPINFO_TOKEN="token from ipinfo"
 ```
+
+If you are using __Twitch__ RTMP URI then in order to view the stream you need to add the `service` prop as `service={"twitch"}` your Twitch username in the `src` prop of the Videostream tag in the Mainstage component `/app/components/conferences/dayOfEvent/mainstage/Mainstage.js`.
+
+Else if you are using __Vimeo__ RTMP URI then then in order to view the stream you need to add the `service` prop as `service={"vimeo"}` your Vimeo live event link in the `src` prop of the Videostream tag in the Mainstage component `/app/components/conferences/dayOfEvent/mainstage/Mainstage.js`. It will be in the format of `https://vimeo.com/webinars/events/{Random-String}/embed`
```

Comment:
Great work on adding the responsiveness, I hope you have done the same for Vimeo also. Please fix the `parentKey`, and we will be good to merge.
Thank you!

Reply:
Yes I have added responsiveness to the Vimeo player as well, in the parent key I have added the check if the NODE_ENV is production the parent key will be the domain on which RC4Conferences is deployed otherwise it will be localhost.